.github/workflows/deploy_prod.yml

```diff
@@ -0,0 +1,44 @@
+name: Deploy to Azure production
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build-and-deploy:
+    runs-on: 'ubuntu-latest'
+
+    steps:
+    - uses: actions/checkout@master
+
+    - uses: azure/docker-login@v1
+      with:
+        login-server: https://leptonregistry.azurecr.io/
+        username: ${{ secrets.ACI_PROD_USERNAME }}
+        password: ${{ secrets.ACI_PROD_PASSWORD }}
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Cache Docker layers
+      uses: actions/cache@v2
+      with:
+        path: /tmp/.buildx-cache
+        key: ${{ runner.os }}-buildx-${{ github.sha }}
+        restore-keys: |
+          ${{ runner.os }}-buildx-
+
+    - name: Build and push
+      id: docker_build
+      uses: docker/build-push-action@v2
+      with:
+        context: ./
+        file: ./Dockerfile
+        builder: ${{ steps.buildx.outputs.name }}
+        push: true
+        tags: leptonregistry.azurecr.io/lepton:latest
+        cache-from: type=local,src=/tmp/.buildx-cache
+        cache-to: type=local,dest=/tmp/.buildx-cache
```

Comment:
Does Azure perform periodic checks for whether latest tag has been updated?

Reply:
Har satt opp continous deployment fra registry til app service slik at den blir oppdatert nå vi pusher til :latest

Reply:
Nice ❤